### ISSUE TITLE
Cannot load AutoML directory.

### ISSUE THREAD

**Issue by Anonumous796** [mljar/mljar-supervised#444]
I want to load the model, but when I try to run the code in the py file, I get an error, but when I do it in the ipnb file, everything works well. Is there any way to fix this?

Code:
```
automl = AutoML(results_path="AutoML_2")
preds = automl.predict(data)
```

Error:

```
2021-07-25 22:24:20,608 supervised.exceptions ERROR Cannot load AutoML directory. '20_LightGBM'
Traceback (most recent call last):
  File "D:\Anaconda_python\lib\site-packages\supervised\base_automl.py", line 191, in load
    ens = Ensemble.load(path, model_subpath, models_map)
  File "D:\Anaconda_python\lib\site-packages\supervised\ensemble.py", line 428, in load
    {"model": models_map[m["model"]], "repeat": m["repeat"]}
KeyError: '20_LightGBM'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:/VKCup/solve.py", line 16, in <module>
    preds = automl.predict(data)
  File "D:\Anaconda_python\lib\site-packages\supervised\automl.py", line 356, in predict
    return self._predict(X)
  File "D:\Anaconda_python\lib\site-packages\supervised\base_automl.py", line 1306, in _predict
    predictions = self._base_predict(X)
  File "D:\Anaconda_python\lib\site-packages\supervised\base_automl.py", line 1238, in _base_predict
    self.load(self.results_path)
  File "D:\Anaconda_python\lib\site-packages\supervised\base_automl.py", line 219, in load
    raise AutoMLException(f"Cannot load AutoML directory. {str(e)}")
supervised.exceptions.AutoMLException: Cannot load AutoML directory. '20_LightGBM'

```

**Comment by pplonski** [comment]
@Anonumous796 thank you for reporting.

Do you have the same LightGBM version in `py` script and in the notebook?

```py
# check the LightGBM version
import lightgbm as lgb
print(lgb.__version__)
```

**Comment by Anonumous796** [comment]
@pplonski Yes, i have 3.2.1 version of lightgbm.
![image](https://user-images.githubusercontent.com/59409403/127052611-539dce8e-10e9-4dea-8801-1dd1e5986696.png)

**Comment by pplonski** [comment]
It was fixed with #395. Closing the issue.

The code with fix is in the master branch. You can try to use it on old results.
